src/main/java/org/corpus_tools/peppermodules/annis/SPointingRelation2ANNISMapper.java

```diff
@@ -115,18 +119,34 @@ public void nodeReached(GRAPH_TRAVERSE_TYPE traversalType,
     // this method behaves exactly as the one in the super class
     super.nodeReached(traversalType, traversalId, currNode, relation, fromNode, order);
     
-    lastEnteredNode = currNode;
+      lastEnteredNode = currNode;
+
+      GlobalIdManager globalIdManager = new GlobalIdManager();
```

Comment:
There should be only one instance of the global ID Manager, you could use
```
this.idManager.getGlobal()
```
or use
```
this.idManager.getEscapedIdentifier(...)
```
instead to the the actual instance.